pkg/tidb/conn.go

```diff
@@ -34,23 +34,19 @@ const (
 	envTidbOverrideEndpointKey = "TIDB_OVERRIDE_ENDPOINT"
 )
 
-func (f *Forwarder) getDBConnProps() (host string, port int, err error) {
-	info, err := f.getServerInfo()
-	if err == nil {
-		host = info.IP
-		port = info.Port
-	}
-	return
+func (f *Forwarder) getDBConnProps() (host string, port int) {
+	return "127.0.0.1", f.tidbPort
```

Comment:
Does this work well in IPv6 envelopment?

Reply:
Not sure about it. Is there any proper way to test this out?

Reply:
@Fullstop000 Start TiDB with parameter:

```bash
./bin/tidb-server -host=[::1]
# or ./bin/tidb-server -host=\[::1\]
```

Then it only listens on ipv6 address.

Reply:
@breeswish PTAL, Shall we handle ipv6 here?

Reply:
According to the doc if you use `tcp4` as listener then only IPv4 address is listened. When using `tcp`, both IPv4 and IPv6 are listened.